wp-content/plugins/core/src/Assets/Theme/Styles.php

```diff
@@ -37,4 +37,13 @@ public function enqueue_styles(): void {
 			wp_enqueue_style( $handle );
 		}
 	}
+
+	/**
+	 * Dequeue WP Core's default block styles from the public FE.
+	 *
+	 * @action wp_enqueue_scripts
+	 */
+	public function dequeue_block_styles(): void {
+		wp_dequeue_style( 'wp-block-library' );
```

Comment:
I need to look at this a little more. My understanding was that core block styles were an opt-in feature. I'll get back to you this afternoon.

Reply:
I was mistaken. There are two separate stylesheets: `wp-block-library` and `wp-block-library-theme`. Only the latter is controlled by the `add_theme_support( 'wp-block-styles' )` directive.

With that said, are you sure you really want to dequeue this, @dpellenwood? What's in there is pretty theme agnostic, and I suspect we'll have more effort replacing it than we would overriding the pieces we don't want.

Reply:
Thanks for checking, Jonathan. Yes, I'm sure. It requires additional code to undo some of the layout & spacing values applied in that file to work with our css/design system. I know we'll end up porting over some of the styles; but, much of it we already support in our system in a different way.